src/app/_components/ArticleRenderer.tsx

```diff
@@ -0,0 +1,16 @@
+import ReactMarkdown from "react-markdown";
+import rehypeRaw from "rehype-raw";
+
+interface MarkdownRendererProps {
+  content: string;
+}
+
+const MarkdownArticle: React.FC<MarkdownRendererProps> = ({ content }) => {
+  return (
+    <div className="prose">
+      <ReactMarkdown rehypePlugins={[rehypeRaw]}>{content}</ReactMarkdown>
```

Comment:
should we support [remark-gfm](https://github.com/remarkjs/remark-gfm)?

Reply:
I added GitHub Flavored Markdown support.